config_example.sh

```diff
@@ -15,3 +15,49 @@
 # Default of ~/.ssh/id_rsa.pub is set in lib/common.sh
 #
 #export SSH_PUB_KEY=~/.ssh/id_rsa.pub
+
+#
+# Select the Container Runtime, can be "podman" or "docker"
+# Defaults to "podman"
+#
+#export CONTAINER_RUNTIME="podman"
+
+#
+# Set the metal3 go path
+#
+#export M3PATH="${GOPATH}/src/github.com/metal3-io"
```

Comment:
Do you expect this to ever be changed?

Reply:
Looks like the script doesn’t let it be changed

Reply:
I don't think they would be changed. Even if we work with forks of the repositories of Metal3, the location should remain the same IMO. Only which repository we actually clone would change. I can anyways parameterize it if you prefer.

Reply:
sorry, those should not be there, they should not be changing indeed.